apps/infrastructure/src/ecs.tf

```diff
@@ -126,3 +129,86 @@ resource "aws_ecs_service" "nextjs_ecs_service" {
   }
 }
 
+data "template_file" "cms_multitenant_app" {
+  for_each = {
+    for key, config in var.websites_configs :
+    key => config
+  }
+  template = file("./task-definitions/cms_app.json.tpl")
+
+  vars = {
+    image                = aws_ecr_repository.strapi_image_repository.repository_url
+    fargate_cpu          = var.cms_app_cpu
+    fargate_memory       = var.cms_app_memory
+    aws_region           = var.aws_region
+    db_host              = aws_rds_cluster.cms_database_cluster.endpoint
+    db_user              = aws_rds_cluster.cms_database_cluster.master_username
+    db_password_arn      = aws_ssm_parameter.cms_database_password.arn
+    bucket_name          = aws_s3_bucket.cms_multitenant_medialibrary_bucket[each.key].bucket
+    admin_jwt_secret_arn = aws_ssm_parameter.cms_admin_jwt_secret.arn
+    db_name              = aws_rds_cluster.cms_database_cluster.database_name
+    db_client            = "postgres"
+    container_port       = var.cms_app_port
+    app_keys             = aws_ssm_parameter.cms_app_keys.arn
+    api_token_salt       = aws_ssm_parameter.cms_api_token_salt.arn
```

Comment:
Shall we use a different token salt per each tenant for better security?

Reply:
done, see last commit